kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyIngressStatusFactory.java

```diff
@@ -16,18 +16,22 @@
 import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxyIngressStatus;
 
 public class KafkaProxyIngressStatusFactory extends StatusFactory<KafkaProxyIngress> {
+
     public KafkaProxyIngressStatusFactory(Clock clock) {
         super(clock);
     }
 
     private KafkaProxyIngress ingressStatusPatch(KafkaProxyIngress observedIngress,
-                                                 Condition condition) {
+                                                 Condition condition,
+                                                 String checksum) {
+
         // @formatter:off
         return new KafkaProxyIngressBuilder()
                 .withNewMetadata()
                     .withUid(ResourcesUtil.uid(observedIngress))
                     .withName(ResourcesUtil.name(observedIngress))
                     .withNamespace(ResourcesUtil.namespace(observedIngress))
+                    .addToAnnotations(ResourcesUtil.DEPENDANT_CHECKSUM_ANNOTATION, checksum)
```

Comment:
Probably we should bow to US English being the global variant of English and spell it `DEPENDENT`.

Or maybe we should use `referents`, since that's what we decided in Robs PR today (it aligns with the use of `*Refs` for the properties).

Reply:
I refuse to concede on British vs US spelling but we should probably align on `referents`